liota/lib/transports/web_socket.py

```diff
@@ -55,7 +55,7 @@ def connect_soc(self):
         try:
             self.WebSocketConnection(self.url, False)
         except Exception:
-            log.exception("WebSocket exception, please check the WebSocket address and try again.")
+            raise Exception("WebSocket exception, please check the WebSocket address and try again.")
```

Comment:
I just noticed that we are losing the complete information about the actual exception, which can be an issue, if we are failing due to some other reason than what your message is suggesting. Here is why it is not a good idea: http://www.ianbicking.org/blog/2007/09/re-raising-exceptions.html